prediction_market_agent_tooling/markets/agent_market.py

```diff
@@ -114,6 +114,16 @@ def get_last_trade_p_no(self) -> Probability | None:
         """
         raise NotImplementedError("Subclasses must implement this method")
 
+    def get_last_trade_yes_outcome_price(self) -> float | None:
+        if last_trade_p_yes := self.get_last_trade_p_yes():
```

Comment:
```suggestion
        # Price on prediction markets are, by definition, equal to the probability of an outcome.
        # Just making it explicit in this function.
        if last_trade_p_yes := self.get_last_trade_p_yes():
```

I was confused for a min why doing this, just a comment for future confusers please 😄